components/centraldashboard-angular/Makefile

```
@@ -2,13 +2,34 @@ IMG ?= ghcr.io/kubeflow/kubeflow/centraldashboard-angular
 TAG ?= $(shell git describe --tags --always --dirty)
 COMMIT = $(shell git rev-parse HEAD)
 
+COMMON_LIB_COMMIT ?= $(shell head -n 1 frontend/COMMON_LIB_COMMIT)
+COMMON_LIB_TARGET_DIR = kubeflow-common-lib
+COMMON_LIB_TMP_ZIP := /tmp/kubeflow-$(COMMON_LIB_COMMIT).zip
+
+.PHONY:fetch-common-lib
+fetch-common-lib:
+	curl -L -f -o $(COMMON_LIB_TMP_ZIP) https://github.com/kubeflow/kubeflow/archive/$(COMMON_LIB_COMMIT).zip
+
+	rm -rf $(COMMON_LIB_TARGET_DIR)
+	mkdir -p $(COMMON_LIB_TARGET_DIR)
+	unzip -q $(COMMON_LIB_TMP_ZIP) "kubeflow-$(COMMON_LIB_COMMIT)/components/crud-web-apps/common/frontend/kubeflow-common-lib/*" -d /tmp
```

`unzip` is not included by default in most linux distros (is available on MacOS)... i can add a check in the `makefile` to "fail fast" if `unzip` missing.. and/or implement logic to store it in a local `bin` direction if we care to...

WDYT?

---

What about using `tar`? We'd need to download the source via this tho
```bash
curl -L -f \
    -o $(COMMON_LIB_TMP_ZIP) \
    https://github.com/kubeflow/kubeflow/archive/$(COMMON_LIB_COMMIT).tar.gz
```

---

This is probably the right choice...

I'm on mac - so by default I tend to avoid the utilities that Mac comes pre-installed with of the BSD flavor that are slightly misaligned with GNU flags...

but in this case - that seems a _lesser evil_ than this implementation... as `unzip` simply may not be there!

i will rework